difflint/data/eslint_terse_reporter.js

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
nit: "Javascript" --> "JavaScript" in commit message.

Reply:
done